ts/components/services/ServiceSectionListComponent.tsx

```diff
@@ -29,6 +29,10 @@ type OwnProps = {
   onRefresh: () => void;
   onSelect: (service: ServicePublic) => void;
   readServices: ReadStateByServicesId;
+  isExperimentalFeaturesEnabled?: boolean;
```

Comment:
Why do you add this prop?
We removed few days ago
https://github.com/teamdigitale/io-app/commit/f05c2fab175fa4fd02615ea303c75892fcc38626#diff-77701a5948e87dab912ec8788cbaab26

It is also never used

Reply:
I don't know why, but I remove it immediately